src/index.ts

```diff
@@ -49,14 +49,16 @@ export function badgen ({
 
   label = sanitize(label)
   status = sanitize(status)
+  const accessibleText = createAccessibleText({label, status})
 
   if (style === 'flat') {
-    return `<svg width="${scale * width / 10}" height="${scale * 20}" viewBox="0 0 ${width} 200" xmlns="http://www.w3.org/2000/svg"${xlink}>
+    return `<svg width="${scale * width / 10}" height="${scale * 20}" viewBox="0 0 ${width} 200" xmlns="http://www.w3.org/2000/svg"${xlink} role="img" aria-label="${sanitizeAttribute(accessibleText)}">
+  <title>${accessibleText}</title>
   <g>
     <rect fill="#${labelColor}" width="${sbRectWidth}" height="200"/>
     <rect fill="#${color}" x="${sbRectWidth}" width="${stRectWidth}" height="200"/>
   </g>
-  <g fill="#fff" text-anchor="start" font-family="Verdana,DejaVu Sans,sans-serif" font-size="110">
+  <g aria-hidden="true" fill="#fff" text-anchor="start" font-family="Verdana,DejaVu Sans,sans-serif" font-size="110">
```

Comment:
Aria-hidden on the visual text prevents IE 11 from reading the text over and over even with a `<title>` and `aria-label`

Reply:
Sweet!